### PR TITLE
Build and test with various versions of flint

### PR DESCRIPTION
**Pull request by saraedum**
fixes https://github.com/wbhart/antic/pull/47

### CODE REVIEW

**Comment by wbhart** [comment]
We are close to Flint 2.7.1. Just letting you know in case this is a priority.

File: .travis.yml [hunk 1]
```diff
@@ -6,11 +6,15 @@ addons:
     packages:
       - texinfo
 env:
-  MAKE="make -j4"
+  MAKEFLAGS=-j2
```
Comment (saraedum) [inline .travis.yml:9]: Travis CI runs on only two vCPUs.

**Comment by saraedum** [comment]
This is not pretty but it seems to do the trick. What do you think @wbhart?

**Comment by wbhart** [comment]
How long will we need to maintain this? Antic may get massively expanded this year, and I'm not that happy about having to do this everywhere. I'm ok with it for now, but is this going to be an ongoing issue?

Also, is it really necessary to replace ${MAKE} with make? This makes it impossible for people on systems with gmake instead of make to use .build_dependencies. Ok, that's probably not a huge number of people, if anyone, but it just seems like a regression. Clearly someone went to the trouble of adding that.

**Comment by saraedum** [comment]
> How long will we need to maintain this? Antic may get massively expanded this year, and I'm not that happy about having to do this everywhere. I'm ok with it for now, but is this going to be an ongoing issue?

I think it would be great to have this until all the usual suspects have upgraded to flint 2.7. A year maybe?

> Also, is it really necessary to replace ${MAKE} with make? This makes it impossible for people on systems with gmake instead of make to use .build_dependencies. Ok, that's probably not a huge number of people, if anyone, but it just seems like a regression. Clearly someone went to the trouble of adding that.

Ok. It was not clear to me that this is used for anything but CI. Let me revert that change then.

**Comment by wbhart** [comment]
Alright, I'll allow this, but I'm not going to maintain it indefinitely.

Also, could the macros possibly be moved out to nf_elem.h so they can be reused elsewhere if necessary? Perhaps place them right at the end of the file?

**Comment by saraedum** [comment]
@wbhart ok. Please have another look.

**Comment by wbhart** [comment]
Looks ok, thanks.

**Pull request merged**